R/fullREMIND.R

```diff
@@ -12,7 +12,6 @@
 #' }
 #'
 fullREMIND <- function() {
-
```

Comment:
I think most of the formatting in this file is on purpose. I personally don't the formatting change, but @LaviniaBaumstark should approve this as well.

Reply:
Oh, that actually was not on purpose. I only wanted to do it on the other files. Should I revert this?